doc/source/examples/resolved-cfd-dem/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -0,0 +1,345 @@
+
+* The ``set frequency`` is set to 1. Since the particle is moving at each time step, the refinement zone around it should be reevaluated at each time step.
+
+* The ``max refinement level`` is set to 6. This parameter limits how small the elements around the particle can get limiting the total number of elements in the problem. Here we limit the mesh size to 48 elements per diameter of the particle. This should be sufficient to get accurate results.
+
+* The ``type`` is set to ``kelly``. Since the particle is moving and we don't want a uniform refinement of all the cell, we use the kelly error estimator based on the ``velocity`` variable.
+
+
+
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection boundary conditions
+	set number                  = 5
+	    subsection bc 0
+		set id = 0
+		set type              = noslip
+	    end
+	    subsection bc 1
+		set id = 1
+		set type              = noslip
+	    end
+
+	    subsection bc 2
+		set id = 2
+		set type              = noslip
+	    end
+	    subsection bc 3
+		set id = 4
+		set type              = noslip
+	    end
+	    subsection bc 4
+		set id = 5
+		set type              = noslip
+	    end
+	end
+
+Here we define the 5 ``no slip`` boundary for all the box walls and let the boundary with ``id=3`` free to represent the top of the box. We refer the reader to the `boundary conditions <../../../parameters.rst>`_. section on how those boundaries are defined. 
+
+.. note:: 
+	The boundary id of dealii rectangular mesh are numbered as such:  :math:`x_{min}=0`, :math:`x_{max}=1`, :math:`y_{min}=2`, :math:`y_{max}=3`, :math:`z_{min}=4`, :math:`z_{max}=5`.
+
+
+Initial condition
+~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection initial conditions
+	  # Type of initial conditionChoices are <L2projection|viscous|nodal>.
+	    set type      = nodal
+	    subsection uvwp
+		    set Function expression = 0; 0; 0
+	    end
+	end 
+
+The initial condition for this case is simple to define. At the start of the simulation, we assume that the particle and the fluid are at rest. From there, we define a uniform velocity field of 0 everywhere. To do that, we used the ``type = nodal`` and then specified a function expression of 0 for all the velocity components.  
+
+Non-Linear Solver Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+	subsection non-linear solver
+	  set verbosity               = verbose
+	  set tolerance               = 1e-6
+	  set max iterations          = 10
+	  set residual precision      = 5
+	  set force rhs calculation   = true
+	end
+	
+* The ``tolerance`` is set to 1e-6. This is small enough to ensure that the flow field is adequately resolved, as here, we expect a velocity of the particle of the order of 10.
+
+* The ``max iterations`` is set to 10. The objective here is to allow enough Newton non-linear steps to ensure the convergence and limit the time pass on a time step if the system is too stiff to reach the tolerance fix.  
```

Comment:
The second sentence is too complicated. Maybe you can break it into two sentences.